app/policies/report_policy.rb

```diff
@@ -14,15 +14,24 @@ def create?
     beis_user?
   end
 
-  def update?
+  def edit?
     beis_user?
   end
 
+  def update?
```

Comment:
Does this need some additional checks around it? A BEIS user can update a submitted report, but a DP user cannot. Similarly, a DP user can update an active report but a BEIS user cannot. It would be good to include some notion of who "owns" the report in each state so that only the current owner is allowed to edit it.